src/mono/wasm/runtime/cwraps.ts

```diff
@@ -291,6 +291,8 @@ export interface t_Cwraps {
     // returns new size of queue after add
     mono_jiterp_tlqueue_add(queue: number, value: VoidPtr): number;
     mono_jiterp_tlqueue_clear(queue: number): void;
+    mono_jiterp_begin_catch(ptr: number): void;
+    mono_jiterp_end_catch(): void;
```

Comment:
this always throws, right ? In that case the return type should be `never` not `void`

Reply:
No, it doesn't always throw